CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [1.0.0]
```

Comment:
Should we add a mention of the change done in #55?

Reply:
Sure, will do! WIll also mention this [one](https://github.com/PicnicSupermarket/diepvries/pull/53)